docs/getting-started/features.md

```diff
@@ -1,6 +1,6 @@
 # Features
 
-After installation, you should see a small icon with the "Vue" logo at the bottom center of your screen. Clicking this(or pressing the "Toggle Vue DevTools" hotkey -- check your startup logs) will open the the Vue DevTools window.
+After installation, you should see a small icon with the "Vue" logo at the bottom center of your screen. Clicking this (or pressing the "Toggle Vue DevTools" hotkey -- check your startup logs) will open the the Vue DevTools window.
```

Comment:
double "the"

Reply:
Thanks for the catch. You'll have to forgive me; I was drunk when I did this and for the entire time since. 

Fixed!

Reply:
no biggy. 
maybe don't get drunk so you don't have to make excuses?
Cheers!

Reply:
Hmm, >hic!< that's a >hic!< pretty good idea! But in the morning I'll be sober, and you, uh, zzzzzzzzz <snore>